src/DetailsView/reports/components/report-sections/common-report-styles.scss

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+@import '../../../../common/styles/colors.scss';
+@import '../../../../common/styles/fonts.scss';
+
+@mixin contentSize {
+    max-width: 960px;
+    margin: 0 auto;
+}
+
+@mixin boxShadow {
+    box-shadow: 0px 0.3px 0.9px rgba(0, 0, 0, 0.108), 0px 1.6px 3.6px rgba(0, 0, 0, 0.132);
+    border-radius: 4px;
+}
+
+body {
```

Comment:
we have a `BodySection` component, so, following the logic here, we should have a proper `body.scss` file too (instead of having mixins and styles on the common file). I'm fine with moving the `body` styles to it's own file in a different PR though

Reply:
Sure. Will do that.